bpf/kmesh/workload/sockops_tuple.c

```diff
@@ -177,9 +177,6 @@ static inline bool conn_from_cni_sim_delete(struct bpf_sock_ops *skops)
 SEC("sockops")
 int record_tuple(struct bpf_sock_ops *skops)
 {
-	// only support IPV4
-	if (skops->family != AF_INET)
```

Comment:
1. kmesh supports only IPv4.
2. In all-zero port listening scenario, socket on the server can process both IPv4 and IPv6 packets. When the server receives an IPv4 packet, the address is mapped to the IPv6 format (IPv4-mapped IPv6 address). However, the family may be AF_INET6. As a result, an error occurs.
Recommended solution:
1. IPv4-mapped IPv6 addresses
![image](https://github.com/kmesh-net/kmesh/assets/85690076/fd0fe80e-ab14-42b6-bf13-3f920fd7d5c6)
if (skops->family != AF_INET) -> if (skops->family != AF_INET || (IPV4-MAPPED_ADDR(skops->local_ip6))

Reply:
good to know.

Can you check only skops->local_ip4?

Reply:
@nlgwcy ITested, the 3rd u32 is 0xfffff0000